packages/extension/src/background/wallet.ts

```diff
@@ -81,6 +81,7 @@ export const PROXY_CONTRACT_CLASS_HASHES = [
   "0x25ec026985a3bf9d0cc1fe17326b245dfdc3ff89b8fde106542a3ea56c5a918",
 ]
 export const ARGENT_ACCOUNT_CONTRACT_CLASS_HASHES = [
+  "0x033434ad846cdd5f23eb73ff09fe6fddd568284a0fb7d1be20ee482f044dabe2",
```

Comment:
Just cos the others don't have leading zero...
```suggestion
  "0x33434ad846cdd5f23eb73ff09fe6fddd568284a0fb7d1be20ee482f044dabe2",
```